### PR TITLE
refactor(rename workflow fastq to rawdata)

### PR DESCRIPTION
**Pull request by ChrOertlin**
## Description

in concert with: https://github.com/Clinical-Genomics/servers/pull/1471

### Added

-

### Changed

-

### Fixed

-

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_cg -t cg -b [THIS-BRANCH-NAME] -a
    ```

### How to test

- [ ] Do ...

### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

File: cg/cli/workflow/fastq/base.py [hunk 1]
```diff
@@ -16,34 +16,41 @@
 
 @click.group(invoke_without_command=True, context_settings=CLICK_CONTEXT_SETTINGS)
 @click.pass_context
-def fastq(context: click.Context):
+def raw_data(context: click.Context):
     """Function for storing fastq-cases"""
```
Comment (diitaz93) [inline cg/cli/workflow/fastq/base.py:20]:
```suggestion
    """Function for storing raw-data cases."""
```

File: cg/cli/workflow/fastq/base.py [hunk 2]
```diff
@@ -16,34 +16,41 @@
...
+    raw_data_service.store_analysis(case_id)
 
 
-@fastq.command("store-available")
+@raw_data.command("store-available")
 @DRY_RUN
 @click.pass_context
-def store_available_fastq_analysis(context: click.Context, dry_run: bool = False):
+def store_available_raw_data_analysis(context: click.Context, dry_run: bool = False):
     """Creates an analysis object in status-db for all fastq cases to be delivered."""
```
Comment (diitaz93) [inline cg/cli/workflow/fastq/base.py:45]:
```suggestion
    """Creates an analysis object in status-db for all raw-data cases to be delivered."""
```

File: tests/cli/workflow/fastq/test_fastq_base.py [hunk 3]
```diff
@@ -1,20 +1,20 @@
...
 
-from cg.cli.workflow.fastq.base import store_available_fastq_analysis, store_fastq_analysis
+from cg.cli.workflow.fastq.base import store_raw_data_analysis, store_available_raw_data_analysis
 from cg.constants.constants import CaseActions, Workflow
 from cg.store.models import Analysis, Case, Sample
 
 
-def test_store_fastq_analysis(another_case_id: str, cli_runner, fastq_context, helpers):
+def test_store_raw_data_analysis(another_case_id: str, cli_runner, fastq_context, helpers):
     """Test for CLI command creating an analysis object for a fastq case"""
```
Comment (diitaz93) [inline tests/cli/workflow/fastq/test_fastq_base.py:10]:
```suggestion
    """Test for CLI command creating an analysis object for a raw-data case."""
```

File: tests/cli/workflow/fastq/test_fastq_base.py [hunk 4]
```diff
@@ -1,20 +1,20 @@
 import logging
 from datetime import datetime
 
-from cg.cli.workflow.fastq.base import store_available_fastq_analysis, store_fastq_analysis
```
Comment (diitaz93) [inline tests/cli/workflow/fastq/test_fastq_base.py:4]: I think `store_available_raw_data_analysis` is not used

File: tests/cli/workflow/fastq/test_fastq_base.py [hunk 5]
```diff
@@ -1,20 +1,20 @@
...
+from cg.cli.workflow.fastq.base import store_raw_data_analysis, store_available_raw_data_analysis
 from cg.constants.constants import CaseActions, Workflow
 from cg.store.models import Analysis, Case, Sample
 
 
-def test_store_fastq_analysis(another_case_id: str, cli_runner, fastq_context, helpers):
+def test_store_raw_data_analysis(another_case_id: str, cli_runner, fastq_context, helpers):
```
Comment (diitaz93) [inline tests/cli/workflow/fastq/test_fastq_base.py:9]: Maybe you can also rename the `fastq_context`

File: tests/cli/workflow/fastq/test_fastq_base.py [hunk 6]
```diff
@@ -25,30 +25,3 @@ def test_store_fastq_analysis(another_case_id: str, cli_runner, fastq_context, h
         )
         > 0
     )
-
-
-def test_store_available_fastq_analysis(
```
Comment (diitaz93) [inline tests/cli/workflow/fastq/test_fastq_base.py:30]: you don't feel this test added some value?

**Review by diitaz93 — APPROVED** [review]
Looks good 👍

File: tests/cli/workflow/fastq/test_fastq_base.py (hunk 6 above)
Comment (ChrOertlin) [reply tests/cli/workflow/fastq/test_fastq_base.py:30]: it was testing whether one cli command invoked the other. that test was not relevant anymore.

File: tests/cli/workflow/fastq/test_fastq_base.py (hunk 5 above)
Comment (ChrOertlin) [reply tests/cli/workflow/fastq/test_fastq_base.py:9]: in a sense it is still a fastq context, as raw-data can be both fastq and now bam. I can specify it being raw-data-fastq

File: cg/cli/workflow/raw_data/base.py [hunk 7]
```diff
@@ -16,34 +16,41 @@
 
 @click.group(invoke_without_command=True, context_settings=CLICK_CONTEXT_SETTINGS)
 @click.pass_context
-def fastq(context: click.Context):
+def raw_data(context: click.Context):
     """Function for storing fastq-cases"""
```
Comment (ChrOertlin) [inline cg/cli/workflow/raw_data/base.py:20]:
```suggestion
    """Function for storing raw-data-cases"""
```

**Pull request merged**